spec/helpers.lua

```diff
@@ -211,14 +211,16 @@ local function wait_until(f, timeout)
     error("arg #1 must be a function", 2)
   end
 
+  ngx.update_time()
```

Comment:
I am thinking we should actually call `ngx.update_time()` prior to each `ngx.time()` call, otherwise we might still be retrieving the same value in the below loop.

Reply:
ngx.sleep updates the time.

Reply:
I wasn't sure, but double check and yeah indeed :+1: